internal/host/refresh_status_preprocessor.go

```diff
@@ -194,8 +193,31 @@ func newValidations(v *validator) []validation {
 			condition: v.sufficientOrUnknownInstallationDiskSpeed,
 			formatter: v.printSufficientOrUnknownInstallationDiskSpeed,
 		},
+		{
+			id:        HasSufficientNetworkLatencyRequirementForRole,
+			condition: v.hasSufficientNetworkLatencyRequirementForRole,
+			formatter: v.printSufficientNetworkLatencyRequirementForRole,
+		}, {
+			id:        HasSufficientPacketLossRequirementForRole,
+			condition: v.hasSufficientPacketLossRequirementForRole,
+			formatter: v.printSufficientPacketLossRequirementForRole,
+		},
 	}
-	return ret
+	var filtered []validation
```

Comment:
Please add a sanity validation that `disabledHostValidations` values are included in `ret` (please change that name to `baseValidations`. Otherwise, I would expect the program to panic

Reply:
Done.